### DISCUSSION TITLE
Google/Apply pay compatibility

### DISCUSSION THREAD

**Discussion by rynfindlay** [breezefront/community#271] [1. Help] [answered]
Hi,

I'm having issues with Apple Pay and Google Pay (Braintree) rendering on the checkout. Are there any compatibility issues or steps required for the external js in this instance?

It is working with Luma and Amasty one step checkout. I am only seeing one error in the console relating to jquery-migrate and no errors in the Magento logs.

Is it possible to disable Breeze Checkout (including layout, styles, js) and use Amasty one step checkout instead as a quick solution?

Thanks for your work on Breeze, it's a great frontend!

<img width="1277" alt="Screenshot 2024-04-29 at 13 35 31" src="https://github.com/breezefront/community/assets/151278823/cb079cdf-f14f-4d85-9434-adccfdf26127">
<img width="680" alt="Screenshot 2024-04-30 at 11 38 03" src="https://github.com/breezefront/community/assets/151278823/3f88608e-1bdb-4a52-af41-35bb02593476">

**Comment by vovayatsyuk** [comment, marked as answer]
Try installing this module: https://github.com/breezefront/module-breeze-paypal-braintree

**Reply by rynfindlay** [reply]
This worked, thank you!

I looked at the module initially but wasn't sure if those minor style updates would resolve.
